stitching/cli/stitch.py

```diff
@@ -315,6 +315,7 @@ def main():
     affine_mode = args_dict.pop("affine")
 
     if affine_mode:
+        args_dict.update(AffineStitcher.AFFINE_DEFAULTS)
         stitcher = AffineStitcher(**args_dict)
```

Comment:
please switch `AffineStitcher` to `Stitcher`, as the use of  `AffineStitcher` is not needed here so we can stay as generic as possible

Reply:
... but only if we DO NOT include a input validation for `AffineStitcher`